src/popover/popover.js

```diff
@@ -442,6 +425,7 @@ class Popover extends React.Component<PopoverPropsT, PopoverPrivateStateT> {
             key="new-layer"
             mountNode={this.props.mountNode}
             onEscape={this.props.onEsc}
+            onDocumentClick={this.onDocumentClick}
```

Comment:
Cool! Now clicks outside will close only the top-most popover. You may want to make this same change for modal and drawer as well. See https://baseweb-git-wpt-5568.uber-ui-platform.now.sh/components/select/#select-in-a-modal-dialog or maybe save to another pr

Reply:
Oh, somehow I missed the modal while searching for usage. I'll add this change to the modal in a follow-up PR.